### PR TITLE
Remove force update of go-licenses binary

### PR DESCRIPTION
**Pull request by dsimansk**
## Description

Remove the force update hack introduced due to `prow-tests:stable` didn't contained the latest `go-licenses` binary. It should ok now, and tests should be passing.

## Changes

* Remove force update of go-licenses binary

### CODE REVIEW

**Comment by dsimansk** [comment]
We're not there yet. There's another PR in go-licenses to fix file permissions. Once another update cycle in `prow-tests:stable` is done, then this PR can be merged to remove the force update.

https://github.com/google/go-licenses/pull/68


/hold

**Comment by rhuss** [comment]
@dsimansk @markusthoemmes I haven't followed the license issue, but I think I just have merged a similar PR from Markus ? Is this PR then not needed anymore ? or does it something different ?

sorry for my ignorance ...

**Comment by markusthoemmes** [comment]
You might want to rebase this. An update has just went through.

@rhuss there was a deadlock essentially with different versions of go-licenses. This workaround by David helped unblock stuff but should eventually become obsolete.

**Comment by dsimansk** [comment]
Ok, so `prow-tests:stable` images was updated, see Slack [thread](https://knative.slack.com/archives/CCSNR4FCH/p1624547784110600?thread_ts=1623781993.092300&cid=CCSNR4FCH).

Once I rebase the PR, it should be pass and good to go to remove the "force update" code.

**Review by navidshaikh — APPROVED** [review]
/lgtm

**Pull request merged**
